Fix/css output

magnificode opened the pull request:
Address issues #11 and #12

saltcod commented:
Hummm trying to sort out what the difference is here. There's no src folder any more? The theme loads the style.min.css everywhere now, with a sourcemap?

magnificode commented:
@saltcod I've removed the src folder from within the `/assets/css/` folders to reduce some of the noise and unnecessary duplicate files.

Now, with this PR, I've added a gulp task that, after the cssnext task compiles the CSS from `/assets/css/` it dumps those flat, non-minified files out into `/dist/`. The cssnano task then takes those flat files, and minifies them. The new task deletes the non-minified CSS files, since the non minified files aren't terribly necessary.

Thanks to the sourcemaps, we really shouldn't need to the non-minified CSS files anymore.

Happy to jump on a call and walk through this as well if that's more helpful!

saltcod commented:
makes perfect sense. That's awesome!
